mlrun/projects/project.py

```diff
@@ -2064,28 +2235,28 @@ def _set_function(
         self.spec.set_function(name, function_object, func)
 
     def remove_function(self, name):
-        """remove a function from a project
+        """remove a function from a project and from the db.
```

Comment:
this is not backwards compatible.
You can add a flag but default must be false.

Reply:
OK but the problem is that if I removing function and when run `list_functions` I will still get the removed function because we returning the list from the db.

@alonmr